src/parse-strict.test.ts

```diff
@@ -57,8 +57,17 @@ describe('parseStrict(string)', () => {
     expect(Number.isNaN(parseStrict('foo'))).toBe(true);
   });
 
-  it('should be case-insensitive', () => {
-    expect(parseStrict('1.5H')).toBe(5400000);
+  it('should be case sensitive', () => {
+    // @ts-expect-error - We expect this to fail.
```

Comment:
Lets be clear in the comments 

```suggestion
    // @ts-expect-error - We expect the types to fail but JS users can still use this 
```